renovate.json

```diff
@@ -21,8 +22,17 @@
         "react-dom",
         "@types/react"
       ],
+      "groupName": "React",
       "groupSlug": "react",
       "rangeStrategy": "replace"
+    },
+    {
+      "matchPackagePatterns": [
```

Comment:
Notice a number of storybook deps in the ignored list. Is that expected?

Reply:
Uh missed that good catch